app/assets/javascripts/darkswarm/services/authentication_service.js.coffee

```diff
@@ -25,9 +25,3 @@ Darkswarm.factory "AuthenticationService", (Navigation, $modal, $location, Redir
     isActive: Navigation.isActive
 
     close: ->
-      if location.pathname in ["/", "/checkout"]
```

Comment:
It does make sense to me to get rid of the annoying redirects from the login modal.